example/mxnet_adversarial_vae/README.md

```diff
@@ -1,22 +1,55 @@
-VAE-GAN in MXNet
+# VAE-GAN in MXNet
 
-Implementation of Autoencoding beyond pixels using a learned similarity metric based on the Tensorflow implementation of https://github.com/JeremyCCHsu/tf-vaegan/
+* Implementation of Autoencoding beyond pixels using a learned similarity metric based on the Tensorflow implementation of https://github.com/JeremyCCHsu/tf-vaegan/
 
-*Please refer to their official Github for details*: https://github.com/andersbll/autoencoding_beyond_pixels
+* Please refer to their official Github for details*: https://github.com/andersbll/autoencoding_beyond_pixels
 
-As the name indicates, VAE-GAN replaces GAN's generator with a variational auto-encoder, resulting in a model with both inference and generation components. 
+* As the name indicates, VAE-GAN replaces GAN's generator with a variational auto-encoder, resulting in a model with both inference and generation components. 
 
-Experiements
+# Experiements
 
-Dataset: caltech 101 silhouettes dataset from https://people.cs.umass.edu/~marlin/data.shtml
+* Dataset: caltech 101 silhouettes dataset from https://people.cs.umass.edu/~marlin/data.shtml
 
-Usage
+# Prerequisites
 
-Using existing models
+* Opencv
+* Python packages required: scipy, scikit-learn and Pillow, opencv python package
 
+# Environment Tested On
+
+Ubuntu Deep Learning AMI, p2.8xlarge
```

Comment:
p2.8xlarge running Amazon EC2 Ubuntu Deep Learning AMI // Can you include the AMI version?